src/js/themes/hpe.js

```diff
@@ -2139,6 +2139,11 @@ const buildTheme = (tokens, flags) => {
         },
       },
     },
+    rangeSelector: {
+      edge: {
+        size: `${primitives.hpe.base.dimension[600]}`, // 24px
```

Comment:
```suggestion
        size: ${primitives.hpe.base.dimension[600]}, // 24px
```
We should be able to do this I think

Reply:
this created another issue so I put back